### DISCUSSION TITLE
Missing Xref

### DISCUSSION THREAD

**Discussion by CesaireMonjoly** [binarly-io/efiXplorer#60] [Q&A] [answered]
I'm completly new to UEFI reverse engineering sorry if the question look stupid I still have a lot to learn
I find some interesting strings in the .rsrc section of a PE32 and IDA don't seems to find any xrefs
So I wonder if this string has any xrefs. btw is the same for all strings in the .rsrc segment of all
I also try to load the PE32 in ghidra, which found xref but it look like ghidra is not sure
![GhidraXref](https://user-images.githubusercontent.com/69987910/184322315-ee60b9ce-e064-49ab-ae16-97d8f3449e73.png)

Did I miss something?

**Comment by xorpse** [comment, marked as answer]
Hi @CesaireMonjoly,

Unfortunately, we don't have the bandwidth to answer general reverse engineering questions. I'd advise seeking help from the [StackExchange](https://reverseengineering.stackexchange.com/) or [Reddit](https://www.reddit.com/r/ReverseEngineering/) reverse engineering communities.

**Reply by CesaireMonjoly** [reply]
Ok thanks :+1: 
I will try on reddit.
